app/assets/javascripts/spree/frontend/spree_payu_integration.js

```diff
@@ -5,7 +5,7 @@
     },
 
     isPayuChoosen: function(e) {
-      return this.choosenPaymentMethodId(e) === this.payuPaymentMethodId();
+      return parseInt(this.choosenPaymentMethodId(e), 10) === parseInt(this.payuPaymentMethodId(), 10);
```

Comment:
Always use `parseInt(whatever, radix)`. it is a good practice, to point to the numeric system you want to parse.

Also, parseInt without this can return weird result sometimes.